SpatialGDK/Documentation/content/glossary.md

```diff
@@ -30,6 +28,15 @@ An Unreal Editor toolbar command which takes a set of Unreal classes and generat
 ### Singleton Actor
 A server-side authoritative Actor that is restricted to one instantiation on SpatialOS. See documentation on [Singleton Actors]({{urlRoot}}/content/singleton-actors.md).
 
+### <ProjectRoot>
+The root folder of your Unreal project. 
+
+### <GameRoot>
+The folder containing your game's `.uproject` and source folder (for example, `<ProjectRoot>/ShooterGame/`). 
```

Comment:
Is "source folder" an Unreal concept, or at least something that Unreal users are likely to understand?

Reply:
Yep

Reply:
Needs to go in the GDoc here: https://docs.google.com/document/d/1YkL5MjhIfM1Mo15TjpvdUyBqKLTbVbrf1MT5iOorLfo/edit

I have added.
Also - there is a discussion about whether to say "game" or "project" as we are flipping between the two and it makes it harder to follow.